extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/security/CSRF.java

```diff
@@ -0,0 +1,153 @@
+         * @return this builder
+         */
+        Builder cookieName(String cookieName);
+
+        /**
+         * The CSRF cookie max age. The default max age are 2 hours.
+         *
+         * @param cookieMaxAge the CSRF cookie max age
+         * @return this builder
+         */
+        Builder cookieMaxAge(Duration cookieMaxAge);
+
+        /**
+         * The CSRF cookie path. The default path is "/".
+         *
+         * @param cookiePath the CSRF cookie path
+         * @return this builder
+         */
+        Builder cookiePath(String cookiePath);
+
+        /**
+         * The CSRF cookie domain.
+         *
+         * @param cookieDomain the CSRF cookie domain
+         * @return this builder
+         */
+        Builder cookieDomain(String cookieDomain);
+
+        /**
+         * Set the 'secure' parameter on the CSRF cookie to 'true' when the HTTP protocol is used.
+         * The cookie will always be secure if the HTTPS protocol is used, even if this method is not called.
+         *
+         * @return this builder
+         */
+        Builder cookieForceSecure();
+
+        /**
+         * Set the HttpOnly attribute to prevent access to the cookie via JavaScript.
+         * The HttpOnly attribute is set by default.
+         *
+         * @param cookieHttpOnly if the HttpOnly attribute should be set
+         * @return this builder
+         */
+        Builder cookieHttpOnly(boolean cookieHttpOnly);
+
+        /**
+         * This method is a shortcut for {@code createTokenPath(Set.of(createTokenPath))}.
+         *
+         * @return this builder
+         * @see #createTokenPath(Set) for more information
+         */
+        Builder createTokenPath(String createTokenPath);
+
+        /**
+         * Create CSRF token only if the HTTP GET relative request path matches one of configured paths.
+         *
+         * @param createTokenPath list of the HTTP GET requests paths for which Quarkus should create a token
+         * @return this builder
+         */
+        Builder createTokenPath(Set<String> createTokenPath);
+
+        /**
+         * Random CSRF token size in bytes. The default size in bytes is 16.
+         *
+         * @param tokenSize the token size in bytes
+         * @return this builder
+         */
+        Builder tokenSize(int tokenSize);
+
+        /**
+         * The CSRF token signature key.
+         *
+         * @param tokenSignatureKey the CSRF token signature key
+         * @return this builder
+         */
+        Builder tokenSignatureKey(String tokenSignatureKey);
+
+        /**
+         * If the CSRF token should be verified by Quarkus. By default, the verification is enabled.
+         * Set {@code false} only if you plan to verify the token manually.
+         *
+         * @param verifyToken if the CSRF token should be verified
+         * @return this builder
+         */
+        Builder verifyToken(boolean verifyToken);
```

Comment:
Sorry @michalvavrik, I missed it, thought it was removed.

IMHO it should be gone, now at least, it is a rather specific case, not really about CSRF prevention properties but where it should be enforced, users who may want to do the manual verification can just use `application.properties`